src/note/commitment.rs

```diff
@@ -33,29 +38,52 @@ impl NoteCommitment {
 impl NoteCommitment {
     /// $NoteCommit^Orchard$.
     ///
-    /// Defined in [Zcash Protocol Spec § 5.4.8.4: Sinsemilla commitments][concretesinsemillacommit].
+    /// Defined in [ZIP-226: Transfer and Burn of Zcash Shielded Assets][notecommit].
     ///
-    /// [concretesinsemillacommit]: https://zips.z.cash/protocol/nu5.pdf#concretesinsemillacommit
-    pub(super) fn derive(
+    /// [notecommit]: https://zips.z.cash/zip-0226#note-structure-commitment
```

Comment:
@str4d ([Link](https://github.com/zcash/orchard/pull/471#discussion_r2550187398)): ```suggestion
    /// [notecommit]: https://zips.z.cash/zip-0226#note-structure-and-commitment
```